python/graphframes/graphframe.py

```diff
@@ -96,6 +97,28 @@ def edges(self):
     def __repr__(self):
         return self._jvm_graph.toString()
 
+    def cache(self):
+        """ Persist the dataframe representation of vertices and edges of the graph with the default
+        storage level.
+        """
+        self._jvm_graph.cache()
+        return self
+
+    def persist(self, storageLevel=StorageLevel.MEMORY_ONLY):
+        """Persist the dataframe representation of vertices and edges of the graph with the given
+        storage level.
+        """
+        javaStorageLevel = self._sc._getJavaStorageLevel(storageLevel)
+        self._jvm_graph.persist(javaStorageLevel)
+        return self
+
+    def unpersist(self, blocking=True):
```

Comment:
Oh!  I was looking at Scala.  I'll ping others about fixing this inconsistency in Spark 2.0: [https://issues.apache.org/jira/browse/SPARK-14717]